### PR TITLE
Login and signup pages

### PR DESCRIPTION
**Pull request by kart-c**
## Made landing and signup pages

CSB links for reference
- Login  https://bp0yz.csb.app/pages/login/login.html
- Signup https://bp0yz.csb.app/pages/signup/signup.html

### CODE REVIEW

File: pages/signup/signup.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+				<h3>Signup</h3>
+				<div class="input-container">
+					<label for="email">Email Address *</label>
+					<input type="email" id="email" name="email" placeholder="Enter your email" />
+				</div>
+				<div class="input-container">
+					<label for="password">Password *</label>
+					<input type="text" id="password" name="password" placeholder="Enter your password" />
+				</div>
+				<span>Forgot password?</span>
```
Comment (VishalPatil18) [inline pages/signup/signup.html]: You could add a `cursor: pointer;` on this as it is also a link or a clickable item.

File: pages/login/login.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+				<h3>Login</h3>
+				<div class="input-container">
+					<label for="email">Email Address *</label>
+					<input type="email" id="email" name="email" placeholder="Enter your email" />
+				</div>
+				<div class="input-container">
+					<label for="password">Password *</label>
+					<input type="text" id="password" name="password" placeholder="Enter your password" />
+				</div>
+				<span>Forgot password?</span>
```
Comment (VishalPatil18) [inline pages/login/login.html:42]: You could add a `cursor: pointer;` on this as it is also a link or a clickable item.

**Review by VishalPatil18 — APPROVED** [review]
Apart from a couple of suggestions, the code looks clean. 
The UI is also quite good. Great work @kart-c 👍

File: pages/signup/signup.html (hunk 1 above)
Comment (kart-c) [reply pages/signup/signup.html]: nice suggestion, thanks Vishal

File: pages/login/login.html (hunk 2 above)
Comment (kart-c) [reply pages/login/login.html:42]: thanks Vishal

**Review by VishalPatil18 — APPROVED** [review]

**Review by Tejinder15 — APPROVED** [review]

**Pull request merged**